permissions/DataStorages.permission

```diff
@@ -5,3 +5,9 @@
 # x-sailjail-description = Removable media
 # x-sailjail-translation-key-long-description =
```

Comment:
Please fill in these values (catalog, key-description, key-long-description).

Reply:
Ah, since this file was already there I just missed that part, will add.

Reply:
Done, so resolving.